client/post-editor/media-modal/image-editor/image-editor-toolbar.jsx

```diff
@@ -66,7 +66,15 @@ const MediaModalImageEditorToolbar = React.createClass( {
 		}
 	},
 
-	renderAspectRatios() {
+	setAspectMenuContext( popoverContext ) {
```

Comment:
Could we clarify the naming here a bit? This function will be given an aspect button reference, correct?